teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
```

Comment:
- [ ] For this inheritance to work you need to import the Person class into this file:

  ```
  require_relative 'person.rb'
  ```